src/app/App/MainScreen/StatusBar/index.tsx

```diff
@@ -45,6 +87,9 @@ export const StatusBar: FC<StatusBarProps> = ({
 				>
 					Import
 				</Button>
+				<Button size="s" view="action" onPress={onExport}>
+					Export
```

Comment:
1. Show progress status on button. Disable it while progress.
2. Move button to another component `ExportNotesButton`